components/aiprompt/views/commands.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Commands View
+page_title: AIPrompt Commands View
+description: Explore the AIPrompt Commands View.
+slug: aiprompt-views-commands
+tags: telerik,blazor,aiprompt,ai,prompt,commands
+published: True
+position: 30
+---
+
+# AIPrompt Commands View
+
+The Commands View displays a set of predefined commands the user can browse and optionally execute. The commands are passed to the component through the `Commands` parameter, which expects a collection of type `List<AIPromptCommandDescriptor>`. Commands can also be organized through a hierarchy, through a parent-child relationship.
+
+>note Commands can be organized through a hierarchy, with a parent-child relationship. Parent commands cannot be directly executed, and only one level of nesting is supported.
```

Comment:
This repeats the previous paragraph.

Reply:
I will leave only the last sentence, as this is really the important thing to note here.